services/src/modules/authentication/anonymous-plugin.ts

```diff
@@ -0,0 +1,24 @@
+import * as http from 'http';
+import { FastifyRequest, FastifyInstance } from 'fastify';
+import * as fp from 'fastify-plugin';
+
+declare module 'fastify' {
+  interface FastifyRequest<
+    HttpRequest = http.IncomingMessage,
+    Query = DefaultQuery,
+    Params = DefaultParams,
+    Headers = DefaultHeaders,
+    Body = DefaultBody
+  > {
+    isAnonymousAccess(): boolean | undefined;
```

Comment:
I don't really see a reason for the convenience function to have a possible return value of undefined for a boolean value, just change the function implementation to `return this._isAnonymousAccess ?? false;`